### PR TITLE
Update/extlibs removed

### PR DESCRIPTION
**Pull request by zsbzsb**
#91 

Also I updated the readme so take a look over that before merging. My english isn't always the best... :tongue:

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -20,17 +20,24 @@ Learn
...
-* The official tutorials (http://www.sfml-dev.org/resources.php#tutorials)
-* The online API documentation (http://www.sfml-dev.org/resources.php#documentation)
+* The official tutorials (http://www.sfml-dev.org/tutorials/)
+* The online API documentation (http://www.sfml-dev.org/documentation/)
 * The community wiki (https://github.com/LaurentGomila/SFML/wiki/)
 * The community forum (http://en.sfml-dev.org/forums/) (or http://fr.sfml-dev.org/forums/ for french people)
 
 Of course, you can also find the SFML.Net API documentation in the SDK.
 
+Building
```
Comment (LaurentGomila) [inline readme.txt]: "Dependencies" would probably be more relevant (see my other comments).

File: readme.txt [hunk 2]
```diff
@@ -20,17 +20,24 @@ Learn
...
+--------
+
+To build and use SFML.NET you must have a copy of CSFML. CSFML can be compiled from
```
Comment (LaurentGomila) [inline readme.txt]: "To run SFML.Net executables", and also add that you need the same runtime dependencies as SFML.

File: readme.txt [hunk 3]
```diff
@@ -20,17 +20,24 @@ Learn
...
+source (https://github.com/LaurentGomila/CSFML/) or downloaded from the offical release
+page (http://www.sfml-dev.org/download/csfml/). Also note that CSFML also relies on main SFML project.
+
```
Comment (LaurentGomila) [inline readme.txt]: I would add a word about carefully matching the versions of CSFML and SFML.Net binaries.

**Comment by zsbzsb** [comment]
Done

**Pull request merged**